sylvia-derive/src/message.rs

```diff
@@ -1203,6 +1157,27 @@ impl<'a> MsgField<'a> {
         }
     }
 
+    pub fn emit_method_field_with_self<GenericT: GetPath>(
```

Comment:
Consider merging with `emit_method_field`.

By default all of the types should be stripped with `Self` and later we might decide to use `AddSelfPath`.